server/Auth.js

```diff
@@ -16,6 +16,18 @@ class Auth {
   constructor() {
   }
 
+  static cors(req, res, next) {
```

Comment:
For what do we require cors to be * ?
They are a security function we are disabling completely here...

Reply:
I spent some more time on this and wasn't able to get the ebooks to work with capacitor/http. There is an open issue and PR being worked on to fix an issue with returning data of the `blob` type for that plugin.

I think this will have to be updated to allow cors for only that `/ebooks` API endpoint and only allow `http://localhost` and `capacitor://localhost` ([described here](https://ionicframework.com/docs/troubleshooting/cors#what-is-cors))